app/src/main/java/org/mozilla/rocket/content/news/NewsPresenter.kt

```diff
@@ -18,8 +18,7 @@ interface NewsViewContract {
     fun updateNews(items: List<NewsItem>?)
 }
 
-class NewsPresenter(private val newsViewContract: NewsViewContract) :
-    ContentPortalView.NewsListListener {
+class NewsPresenter(private val newsViewContract: NewsViewContract) : NewsListListener {
```

Comment:
nit: please move parameters in new line

Reply:
Since it can fit on a single line, it may not need to move parameters in another line.
https://developer.android.com/kotlin/style-guide#functions